app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,109 @@
+# TASKS = ["Watch cs fun video", "Do the laundry", "Answer 2 emails"]
+
+class TasksController < ApplicationController
+    def index
+        @tasks = Task.all
+    end
+
+    def show 
+        task_id = params[:id]
+        
+        begin 
+            @task = Task.find(task_id)
```

Comment:
Instead of using `find`, consider using `find_by`. `find_by` return nil if the object is not found.